src/shapes/sprite.js

```diff
@@ -14,16 +14,18 @@ function SpriteShape(params) {
     dx: params.x,
     dy: params.y,
     speed: properties.speed,
-    animation: null
+    update: null,
+    animation: properties.animation === true ? true : false,
```

Comment:
"teste" in js is `true`, right?
Since we write test for all cases (Fall and success case, there's no problem with that)